edb/pgsql/parser/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.so
+psqlparse/parser.c
```

Comment:
This `.gitignore` seems to be unnecessary, because all ignored files are covered by the top-level `.gitignore` (and `./build` is no longer produced).